docs/src/index.md

```diff
@@ -29,6 +29,8 @@ Development of ITensor is supported by the Flatiron Institute, a division of the
 
 ## News
 
+- April 16, 2024: ITensors.jl v0.4 has been released. This version removes HDF5 as a dependency (a breaking change) and moves the HDF5 read and write functions for ITensor, MPS, MPO, and other associated types into a package extension. To enable ITensor HDF5 features, install the HDF5 package and put `using HDF5` before `using ITensors` in your code. Other recent changes include support for multiple GPU backends using package extensions and support for the Observers.jl package (also through a package extension).
```

Comment:
The ordering of calling `using HDF5` and `using ITensors` shouldn't matter.

Also, is `Observers` actually supported in `ITensors.dmrg`? I don't think it is right now, that sounds misleading.

Reply:
Please see if the update text is more accurate